packages/@coorpacademy-components/src/molecule/hero/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import get from 'lodash/fp/get';
+import Button from '../../atom/button';
+import Provider from '../../atom/provider';
+import style from './style.css';
+
+const Hero = (props, context) => {
+  const {skin} = context;
+  const {image, title, author, progress, onClick, submitValue} = props;
+
+  const primaryColor = get('common.primary', skin);
+  const cardStyle = classnames(style.hero, title ? null : style.lazy);
+
+  const inlineProgressValueStyle = {
+    backgroundColor: primaryColor,
+    width: `${progress * 100}%`
+  };
+
+  return (
+    <div className={cardStyle} data-name="hero">
+      <div className={style.imageWrapper}>
+        <div
+          data-name="cover"
+          className={style.image}
+          style={{
+            backgroundColor: primaryColor,
+            backgroundImage: image ? `url('${image}')` : 'none'
+          }}
+        />
+      </div>
+      <div data-name="info" className={style.infoWrapper}>
```

Comment:
should follow analysis around splitting `card` component to have a `card-content` and use it here to avoid having 2 sources of same UI (card content, without borders radius, shadow, etc)